### ISSUE TITLE
Dynmap Beta5-431 Startup Initialization Error

### ISSUE THREAD

**Issue by ThorinEk** [webbukkit/dynmap#3180]
**Issue Description:**
Dynmap fails to initialize when the server starts.

* **Dynmap Version:**
Dynmap-3.1-beta5-spigot.jar
* **Server Version:**
Paper version git-Paper-257 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)

* **Pastebin of Configuration.txt:**
https://paste.ubuntu.com/p/jFRJsGZdmr/
* **Server Host *
ApexMinecraftHosting
* **Pastebin of crashlogs or other relevant logs:**
https://paste.ubuntu.com/p/WXq7mxWmj6/
* **Steps to Replicate:**
1. Start the server
 [ ] *I have looked at all other issues and this is not a duplicate* 
Yes
 [ ] *I have been able to replicate this*
It happens on every startup.

**Comment by Baktus79** [comment]
I have the same issue.
```
10.11 16:31:32 [Server] INFO [dynmap] Enabling dynmap v3.1-beta5-431*
10.11 16:31:33 [Server] INFO [dynmap] Added 4 custom biome mappings
10.11 16:31:33 [Server] WARN [dynmap] Loaded class ru.tehkode.permissions.bukkit.PermissionsEx from PermissionsEx v1.23.4 which is not a depend, softdepend or loadbefore of this plugin.
10.11 16:31:33 [Server] INFO [dynmap] Using PermissionsEx 1.23.4 for access control
10.11 16:31:33 [Server] ERROR Error occurred while enabling dynmap v3.1-beta5-431 (Is it up to date?)
10.11 16:31:33 [Server] INFO java.lang.NullPointerException: null
10.11 16:31:33 [Server] INFO at org.dynmap.DynmapCore.findExecutableOnPath(DynmapCore.java:441) ~[?:?]
10.11 16:31:33 [Server] INFO at org.dynmap.DynmapCore.enableCore(DynmapCore.java:472) ~[?:?]
10.11 16:31:33 [Server] INFO at org.dynmap.bukkit.DynmapPlugin.doEnable(DynmapPlugin.java:961) ~[?:?]
10.11 16:31:33 [Server] INFO at org.dynmap.bukkit.DynmapPlugin.onEnable(DynmapPlugin.java:941) ~[?:?]
10.11 16:31:33 [Server] INFO at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:239) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-261]
10.11 16:31:33 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
10.11 16:31:33 [Server] INFO [dynmap] Disabling dynmap v3.1-beta5-431
10.11 16:31:33 [Server] INFO [dynmap] Unloaded 0 components.
10.11 16:31:33 [Server] INFO [dynmap] Disabled
```

Using Paper 1.16.4

**Comment by Nogapra** [comment]
I have the same issue with PaperMC 282 (MC 1.16.4)

I specify that my MAP files are stored in SQL and that I did not test with a brand new version, my card was generated with a 1.16.3 version of paper

Full Log : https://pastebin.com/6eQG90GF

![image](https://user-images.githubusercontent.com/42911787/99540214-88702a80-29af-11eb-88b8-a5b7fcaa1eb6.png)

**Comment by MCAlagaesia** [comment]
@Baktus79  @Nogapra You both seem to be encountering a different issue, an NPE in findExecutableOnPath. Check out #3170 instead.

**Comment by broken1arrow** [comment]
I get this error too, will test that you link if it work.

````
[14:00:27] [Server thread/INFO]: [dynmap] Opening MySQL database localhost:3306/Midgardsaga as map store
[14:00:27] [Server thread/ERROR]: Error occurred while enabling dynmap v3.1-beta5-431 (Is it up to date?)
java.lang.NullPointerException: null
	at org.dynmap.DynmapCore.findExecutableOnPath(DynmapCore.java:441) ~[?:?]
	at org.dynmap.DynmapCore.enableCore(DynmapCore.java:472) ~[?:?]
	at org.dynmap.bukkit.DynmapPlugin.doEnable(DynmapPlugin.java:961) ~[?:?]
	at org.dynmap.bukkit.DynmapPlugin.onEnable(DynmapPlugin.java:941) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.4.jar:git-Paper-296]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.4.jar:git-Paper-296]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.4.jar:git-Paper-296]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:501) ~[patched_1.16.4.jar:git-Paper-296]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:415) ~[patched_1.16.4.jar:git-Paper-296]
	at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.4.jar:git-Paper-296]
	at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:239) ~[patched_1.16.4.jar:git-Paper-296]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:940) ~[patched_1.16.4.jar:git-Paper-296]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-296]
	at java.lang.Thread.run(Thread.java:834) [?:?]
[14:00:27] [Server thread/INFO]: [dynmap] Disabling dynmap v3.1-beta5-431

```

**Comment by MCAlagaesia** [comment]
@broken1arrow That is a different issue, you can check it out at #3170.

**Comment by ThorinEk** [comment]
An update on this issue -  the startup error still persists, however I solved it by switching back to **dynmap v3.1-beta4-421**. This would suggest the problem is only with the newest version, also the problem started when I updated to **dynmap Beta5-431**. While it is great that the old version still works, I fear eventual trouble when I will be forced to update due to a Minecraft update for an example.

**Comment by FedUpWith-Tech** [comment]
Closing due to age
